phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/ControllerConfiguration.java

```diff
@@ -143,4 +143,8 @@ public SettingsController settingsController() {
                 settingsConfiguration.settingsManager()
         );
     }
+
+    public String getWineEnginesPath() {
+        return this.wineEnginesPath;
```

Comment:
I think we should not consider special engine types on this level. I would prefer to build the specific engine path based on `application.user.engines` later on.